Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/AjuntamentdeBarcelona/decidim
-  revision: 073a850a51f0a8110a9ffdfd5402beeaa868f606
+  revision: 4a3fb9e582878b882d635795cec010b222ecfd1f
```

Comment:
Revision here shouldn't change as it must be the latest in the `0.26-stable-bcn` branch from https://github.com/AjuntamentdeBarcelona/decidim/